lib/smart-app.js

```diff
@@ -58,7 +58,12 @@ module.exports = class SmartApp {
 		this._apiUrl = options.apiUrl ? options.apiUrl : 'https://api.smartthings.com'
 		this._refreshUrl = options.refreshUrl ? options.refreshUrl : 'https://auth-global.api.smartthings.com/oauth/token'
 
-		this._authorizer = new Authorizer(options)
+		this._authorizer = new Authorizer({
```

Comment:
This change is just a simple way to also pass in the SDK's logger to bubble up errors that come from the `HttpKeyResolver`